README.md

```diff
@@ -15,9 +15,26 @@ Json schemas for types used in the contract apis can be generated by navigating
 directory and running `cargo schema`. This will generate a `./contracts/{contract_name}/schema` directory containing the
 json schemas for types used to instantiate, execute, query etc.
 
+### Initial Setup
+Make sure to add `--recurse-submodules` flag when cloning:
+```bash
+git clone --recurse-submodules  https://github.com/axelarnetwork/axelar-amplifier.git
+```
+so you automatically initialize and update all submodules during the initial clone. If you've already cloned the repository without initializing submodules and encounter errors like:
+
+```
+Error: Custom { kind: Other, error: "protoc failed: Could not make proto path relative: proto-files/ampd/v1/ampd.proto: No such file or directory" }
+```
+
+Run the submodule initialization commands as shown below:
+```bash
+git submodule init
+git submodule update
+```
```

Comment:
what about when pulling from remote once the repo was already initialized correctly, I think `git pull` supports `--recurs-submodules` as well? Document how commits should be fetched